lib/protocol/errors.js

```diff
@@ -804,16 +804,20 @@ function getResponseForW3CError (err) {
   if (isErrorType(err, errors.BadParametersError)) {
```

Comment:
why this condition is needed?

Reply:
JSONWP are almost all 500, except for `BadParameters`, `NotYetImplemented` and `NoSuchDriver`.

Reply:
this might be handy as a comment